mix.exs

```diff
@@ -157,7 +157,9 @@ defmodule Timber.Mixfile do
       {:earmark, "~> 1.0", only: [:dev, :docs]},
       {:ecto, "~> 2.0", optional: true},
       {:ex_doc, "~> 0.14", only: [:dev, :docs]},
+      {:hackney, "~> 1.6", optional: true},
       {:excoveralls, "~> 0.5", only: [:test]},
+      {:msgpax, "~> 1.0"},
```

Comment:
Thoughts on if this should be optional as well? It felt excessive in the README.